wgpu-native/src/swap_chain.rs

```diff
@@ -197,13 +202,15 @@ pub fn swap_chain_get_next_texture<B: GfxBackend>(
         ref_count,
     });
 
-    SwapChainOutput { view_id }
+    Ok(SwapChainOutput { view_id })
 }
 
 #[cfg(feature = "local")]
 #[no_mangle]
 pub extern "C" fn wgpu_swap_chain_get_next_texture(swap_chain_id: SwapChainId) -> SwapChainOutput {
-    gfx_select!(swap_chain_id => swap_chain_get_next_texture(&*GLOBAL, swap_chain_id, PhantomData))
+    gfx_select!(swap_chain_id => swap_chain_get_next_texture(&*GLOBAL, swap_chain_id, PhantomData)).unwrap_or_else(|_| SwapChainOutput {
```

Comment:
given that this is lightweight, just `unwrap_or` is fine, no need to introduce a closure